src/__tests__/__snapshots__/TipBox.spec.js.snap

```diff
@@ -0,0 +1,1179 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`TipBox having custom background color renders correctly 1`] = `
+<div
+  class="alert box-container alert-default"
+  style="background-color: white; border-color: white;"
+>
+  <!---->
+   
+  <!---->
+   
+  <div
+    class="box-body-wrapper"
+  >
+    <div
+      class="icon-wrapper"
+    >
+      <span />
+    </div>
+     
+    <!---->
+     
+    <div
+      class="contents"
+    />
+     
+    <button
+      aria-label="Close"
+      class="close"
+      data-dismiss="alert"
+      style="display: none;"
+      type="button"
+    >
+      <span
+        aria-hidden="true"
+      >
+        ×
+      </span>
+    </button>
+  </div>
+</div>
+`;
+
+exports[`TipBox having custom border color renders correctly 1`] = `
+<div
+  class="alert box-container alert-default"
+  style="border: 1px solid grey;"
+>
+  <!---->
+   
+  <!---->
+   
+  <div
+    class="box-body-wrapper"
+  >
+    <div
+      class="icon-wrapper"
+    >
+      <span />
+    </div>
+     
+    <!---->
+     
+    <div
+      class="contents"
+    />
+     
+    <button
+      aria-label="Close"
+      class="close"
+      data-dismiss="alert"
+      style="display: none;"
+      type="button"
+    >
+      <span
+        aria-hidden="true"
+      >
+        ×
+      </span>
+    </button>
+  </div>
+</div>
+`;
+
+exports[`TipBox having custom border left color renders correctly 1`] = `
+<div
+  class="alert box-container alert-default"
+  style="border-left: 5px solid blue;"
+>
+  <!---->
+   
+  <!---->
+   
+  <div
+    class="box-body-wrapper"
+  >
+    <div
+      class="icon-wrapper"
+    >
+      <span />
+    </div>
+     
+    <!---->
+     
+    <div
+      class="contents"
```

Comment:
is there some way to configure the output to be more concise in such cases? ( `<div class="contents">` )

Reply:
>  Jest uses pretty-format to make snapshots human-readable during code review
[Link](https://jestjs.io/docs/en/snapshot-testing)

I was thinking this too tho I prefer the pretty format tho. I understand that the more concise output is more accurate to what we see in devTool etc.

Reply:
It seems that the snapshot serializer I'm using `jest-serializer-vue` makes it pretty format, I can't change it otherwise. And there seems to be no serializers that **don't** use pretty format

Reply:
hmm ok. iirc storybook's serialization has this, but we can deal with that later on